src/libs/ajax.js

```diff
@@ -235,6 +235,11 @@ const User = signal => ({
     }
   },
 
+  getTos: async () => {
+    const response = await fetchSam('tos/text', _.merge(authOpts(), { signal, method: 'GET' }))
+    return await response.text()
```

Comment:
because it's already a promise, and this whole function will be awaited:
```suggestion
    return response.text()
```